src/client/kotlin/tech/thatgravyboat/skyblockapi/api/events/screen/InventoryChangeEvent.kt

```diff
@@ -17,4 +19,16 @@ class InventoryChangeEvent(
 ) : SkyBlockEvent() {
     val isInPlayerInventory = slot.container is Inventory
     val title = titleComponent.stripped
+    val itemStacks = inventory.map { it.item }
+
+    val isSkyBlockFiller = item.isEmpty || item in ItemTag.GLASS_PANES
+
+    val isInTopRow = slot.index < 9
+    val isInBottomRow = (screen as? ContainerScreenAccessor)?.containerRows?.let { (slot.index - 1) >= it * 9 } ?: false
+    val isOnLeftColumn = slot.index % 9 == 0
+    val isOnRightColumn = slot.index % 9 == 8
+
+    val isOnSides = isOnLeftColumn || isOnRightColumn
+    val isInTopRowOrBottomRow = isInTopRow || isInBottomRow
+    val isInMainPart = !isOnSides || !isInTopRowOrBottomRow
```

Comment:
this should be !isOnSides && !isInTopRowBottomRow or else itd fail on slot index 1 as its not on the sides and will return true but its actually in the top row